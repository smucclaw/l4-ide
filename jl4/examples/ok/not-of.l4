#EVAL NOT TRUE
#EVAL NOT NOT TRUE
#EVAL NOT OF TRUE
#EVAL NOT OF NOT OF TRUE
