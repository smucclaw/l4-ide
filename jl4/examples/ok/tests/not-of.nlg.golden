not `TRUE`
not not `TRUE`
not `TRUE`
not not `TRUE`
